### PR TITLE
Final new seed data

### LINKED ISSUE #83: Make more seed data
We'd like to make testing easier by having more seed data.

Please add 10-30 additional participants in different states.
- many citations 
- no citations
- done status
- new status

### PR DESCRIPTION
**Pull request by xleung26**
Fixes #83 
Added 31 data set of participants and citations. Check pgAdmin to see data. May have conflicts between fields in interface vs fields in the backend. However, the function to generate data works.

### CODE REVIEW

File: main-app/client/src/components/ParticipantsList/ParticipantsList.js [hunk 1]
```diff
@@ -12,6 +12,12 @@ class Participants extends Component {
     curPage: 1,
     paginationStartPoint: 1,
     loading: false,
+<<<<<<< HEAD
```
Comment (thomasjinlo) [inline main-app/client/src/components/ParticipantsList/ParticipantsList.js]: We should remove this.

File: main-app/client/src/components/ParticipantsList/ParticipantsList.js [hunk 2]
```diff
@@ -12,6 +12,12 @@ class Participants extends Component {
...
+    records: participantData,
+    participants: [],
+   
+  };
+=======
```
Comment (thomasjinlo) [inline main-app/client/src/components/ParticipantsList/ParticipantsList.js]: ditto

File: main-app/client/src/components/ParticipantsList/ParticipantsList.js [hunk 3]
```diff
@@ -49,6 +55,7 @@ class Participants extends Component {
       },
     },
   }
+>>>>>>> 69b4843156929e9f911b5d423a55b6db0ab397e3
```
Comment (thomasjinlo) [inline main-app/client/src/components/ParticipantsList/ParticipantsList.js]: last one.

**Comment by abregorivas** [comment]
This fails when run with docker-compose up --build.
Can you test to make sure that this runs during the build. 
Please note I removed the merge conflicts prior to doing the build. 

image

**Comment by xleung26** [comment]
@abregorivas Hey, I see that you were busy today. I checked out my branch and @ultamatt checked out my branch. docker-compose up --build worked for both of us, so I think it may be that instance you were trying that it failed (maybe prune all containers and images and try again?). Can we work on updating the new fields next week? Thanks!

**Comment by thomasjinlo** [comment]
@xleung26 Can you rebase from latest master build and try again? I'm running into an issue with building docker as well.

**Pull request closed without merging**